crates/math/src/uint.rs

```diff
@@ -38,19 +39,26 @@ where
     pub const fn number(&self) -> U {
         self.0
     }
+
+    pub const fn number_ref(&self) -> &U {
+        &self.0
+    }
```

Comment:
I think the more idiomatic way for defining these two functions is:

```rust
// takes a ref, return a ref
pub const fn number(&self) -> &U {
    &self.0
}

// takes an owned value, return an owned value
pub const fn into_number(self) -> U {
    self.0
}
```

Let's leave this to a later PR to fix.